packages/eslint-plugin/src/rules/restrict-plus-operands.ts

```diff
@@ -5,6 +5,7 @@ import * as util from '../util';
 type Options = [
   {
     checkCompoundAssignments?: boolean;
+    allowAny?: boolean;
   },
 ];
 type MessageIds = 'notNumbers' | 'notStrings' | 'notBigInts';
```

Comment:
Yeah, this message ID's usage has drifted from its original intent a bit. It'd probably be better to call it something like `notNumbersOrStrings`.

Not a blocker for this PR if you ask me 🙂 but optional if you're interested!

Reply:
since it's already merged... I just made another pr for this one #4332